packages/terra-clinical-onset-picker/README.md

```diff
@@ -18,7 +18,7 @@ The terra-clinical-onset-picker component provides users a way to enter or selec
 
 This component supports locales defined in the Terra component standards, except Arabic.
```

Comment:
@mjhenkes Should we log an issue to add Arabic so all terra-components meet the same translation standards?

Reply:
We don't actually support Arabic, (https://github.com/cerner/terra-aggregate-translations/blob/master/config/i18nSupportedLocales.js) even though we say we do (https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#internationalization-i18n-support)
My opinion is to remove the mentions to arabic from here and terra-ui.

Reply:
```suggestion
This component supports locales defined in the Terra component standards.
```

Reply:
Ah right, we removed Arabic support a while back: https://github.com/cerner/terra-aggregate-translations/pull/2#issue-274424666